src/api/pull/index.ts

```diff
@@ -86,11 +86,18 @@ export async function listOwners(
     owner,
     repo,
   });
-  const collaborators = collaboratorInfos.map((c) => {
-    return {
-      githubName: c.login,
-    };
-  });
+
+  const collaborators = collaboratorInfos
+    .filter((collaboratorInfo) => {
+      return (
+        collaboratorInfo.permissions.admin || collaboratorInfo.permissions.push
```

Comment:
Better add some comments.

Reply:
Good Idea.

Reply:
I think maybe better move this logic into service.

Reply:
The api only gather the input and response the output. So I think this should be in service.

Reply:
But why we call the `github.repos.listCollaborators` API in the api layer, in fact, the collaborator info is only used in the case of the pull request without member-list file.

Reply:
Because I think if we get GitHub information for different situations, it may be very complicated.